.github/workflows/closing-pr.yml

```diff
@@ -0,0 +1,26 @@
+name: Closing PR
+
+on:
+  pull_request_target:
```

Comment:
`pull_request_target` is used because this workflow needs write permission to add a label.